cfripper/model/result.py

```diff
@@ -138,6 +138,10 @@ def add_failure_blocking_rule(self, failure: Failure):
     def valid(self) -> bool:
         return not bool([rule for rule in self.failed_rules if rule.rule_mode == RuleMode.BLOCKING])
 
+    @property
+    def reason(self) -> str:
```

Comment:
I'm not sure about this. It should be reasons, and we may want optionally monitored rules too.